### ISSUE TITLE
HTML elements need lang attribute

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15189]
Many of our HTML elements (for all the pages) are lacking a [lang attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang).

**Comment by garrett** [comment]
Clarification: I'm talking about this:
```html
<html lang="FOO">
```

Not various HTML elements in general. :laughing:

**Comment by marusak** [comment]
All `<html>` elements in iframes have this set up. The only place I can see this missing is on shell (the top most one).
Login page is fine as well.

Or have you found more cases where this is not working? @garrett

**Comment by marusak** [comment]
I sent https://github.com/cockpit-project/cockpit/pull/15198. Now I think all places are addressed, but since you said `HTML elements` I guess there are more places?

**Comment by garrett** [comment]
The a11y tool complained about it missing for every single page. (Perhaps it was really complaining about the shell?)
